app/services/dismiss_topics.rb

```diff
@@ -17,6 +18,7 @@ def rows
       .joins("LEFT JOIN topic_users ON topic_users.topic_id = topics.id AND topic_users.user_id = #{@user.id}")
       .where("topics.created_at >= ?", since_date)
       .where("topic_users.id IS NULL")
+      .where("topics.archetype <> ?", Archetype.private_message)
```

Comment:
I realized that in previous commits I forgot about private messages. No big harm as DismissedTopicUsers table is regularly cleaned